web/src/app/main/components/ToolbarTitle.js

```diff
@@ -45,10 +45,10 @@ const mapSingular = {
 
 const nameQuery = typeName => gql`
   query($id: ID!) {
-    data: ${typeName}(id: $id) {
-      id
-      name
-    }
+  data: ${typeName}(id: $id) {
```

Comment:
nit: revert spacing here (might need to run both linters as i saw some spacing in the go code not match as well)